### PR TITLE
Making windsor look like core part 1

### PR DESCRIPTION
**Pull request by fir3pho3nixx**
As per #181 I have cleaned up and implemented the following: 

 - Renamed License.txt to LICENSE
 - Renamed Changes.txt to CHANGELOG.md

Will follow up with another PR for the BreakingChanges.txt after merging it into CHANGELOG.md same as Castle.Core.

### CODE REVIEW

File: buildscripts/Castle.Windsor.nuspec [hunk 1]
```diff
@@ -29,14 +29,14 @@
...
-    <file src="License.txt" />
+    <file src="CHANGELOG.md" />
+    <file src="LICENSE" />
     <file src="build/NET35/NET35-Release/pkg/bin/Castle.Windsor.???" target="lib/net35" />
     <file src="build/NET40/NET40-Release/pkg/bin/Castle.Windsor.???" target="lib/net40" />
     <file src="build/NET40CP/NET40CP-Release/pkg/bin/Castle.Windsor.???" target="lib/net40-client" />
     <file src="build/NET45/NET45-Release/pkg/bin/Castle.Windsor.???" target="lib/net45" />
     <file src="buildscripts/ASL - Apache Software Foundation License.txt" />
-    <file src="buildscripts/readme.txt" />
+    <file src="buildscripts/README.md" />
```
Comment (jonorossi) [inline buildscripts/Castle.Windsor.nuspec]: I'm pretty sure this little `readme.txt` file is explicitly for display when the Windsor package is installed by NuGet as it'll get displayed:
- https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#adding-a-readme-and-other-files

Comment (fir3pho3nixx) [inline buildscripts/Castle.Windsor.nuspec]: Did not know this was a thing.

**Comment by fir3pho3nixx** [comment]
@jonorossi - the commit for the nuspec breaking change todo with Readme.txt has been removed.

**Comment by jonorossi** [comment]
👍

**Pull request merged**